frontend/src/metabase/api/dashboard.ts

```diff
@@ -126,7 +150,10 @@ export const dashboardApi = Api.injectEndpoints({
           body,
         }),
         invalidatesTags: (_, error) =>
-          invalidateTags(error, [listTag("dashboard")]),
+          invalidateTags(error, [
+            listTag("dashboard"),
+            tag("parameter-values"),
```

Comment:
This is more of a general question: can we scope the `parameter-values` invalidation to only the ones for the dashboard that is being edited/deleted?

Reply:
yes, but I think in practice it would be the same, unless you edit multiple dashboards in a row